.travis.yml

```diff
@@ -10,7 +10,7 @@ before_install:
 
 script:
   - coala --non-interactive -V
-  - npm run build
```

Comment:
Can we please keep the build process in .travis.yml , not in a script called `deploy.sh` .